src/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { memo } from 'react';
+import cn from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const Navigation: React.FC = memo(() => (
+  <div className="navbar-brand">
+    <NavLink
+      to="/"
+      className={({ isActive }) => cn('navbar-item',
+        { 'is-active': isActive })}
+    >
+      Home
+    </NavLink>
+
+    <NavLink
+      to="tabs"
+      className={({ isActive }) => cn('navbar-item',
+        { 'is-active': isActive })}
```

Comment:
it would be better to create components and pass way and title to them, like this:
`       <NavItem to="/" title="Home" /> 
<NavItem to="/tabs" title="Tabs" />`
But it's up to you. Your version is ok too.